fvm/src/call_manager/default.rs

```diff
@@ -36,6 +36,8 @@ pub struct InnerDefaultCallManager<M: Machine> {
     machine: M,
     /// The gas tracker.
     gas_tracker: GasTracker,
+    /// The gas premium paid by this message.
+    gas_premium: TokenAmount,
```

Comment:
Maybe put this in the gas tracker?
Looks kinda out of place.

Reply:
oh it was and you moved it out.
Any particular reason?

Reply:
Note i am fine with it, i shall blame raul's insistence to hide it on how it ended there in the first place (git will blame me).

Reply:
Yeah, basically, it has nothing to do with gas tracking. It's used for gas _charging_ (which is why the gas tracker never even looked at it). Same with the base fee.